.github/workflows/scala.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           ${{ runner.os }}-${{matrix.scala}}-${{ hashFiles('**/*.sbt') }}
           ${{ runner.os }}-${{matrix.scala}}-
     - name: Run tests
-      run: sbt ${{matrix.sbt-args}} ++${{ matrix.scala }} test
+      run: sbt ${{matrix.sbt-args}} "set crossScalaVersions := Seq(\"${{ matrix.scala }}\")" "set \`domains-catsJVM\` / crossScalaVersions := Seq(\"${{ matrix.scala }}\")" "set \`domains-catsJS\` / crossScalaVersions := Seq(\"${{ matrix.scala }}\")" "++${{ matrix.scala }} test"
```

Comment:
```suggestion
      run: sbt ${{matrix.sbt-args}} ++${{ matrix.scala }} test
```

Reply:
`crossScalaVersions` seems unnecessary

Reply:
It's the result of a compromise.
See 
https://github.com/xarcher/dsl-domains-cats/pull/2
https://github.com/xarcher/dsl-domains-cats/pull/3
It seems that sbt now only can `++` a existing crossScalaVersion.

Reply:
Then just add an `!`, please

Reply:
Thanks for the suggestion. Find the answer in
https://github.com/sbt/sbt/issues/3698#issuecomment-340876280
Already accepted the change. See
https://github.com/xarcher/dsl-domains-cats/actions/runs/3166811897

Reply:
Sorry for my ignorance of the pull request function.😥